packages/cli/.gitignore

```diff
@@ -1,3 +0,0 @@
-coverage
-node_modules
```

Comment:
do we need a line for `cli/node_modules` in the top-level as well?

Reply:
I don't think so. `node_modules` in top level `.gitignore` should have it covered.

Reply:
top level has a `node_modules` folder which works for all sub-directories. `sandbox` was specific to cli so it was marked as such.

Reply:
oops didn't see your response there ... but what @raymondfeng said basically.

Reply:
👍